app/models/manageiq/providers/azure/cloud_manager.rb

```diff
@@ -46,4 +46,30 @@ def verify_credentials(_auth_type = nil, options = {})
 
     true
   end
+
+  # Operations
+
+  def vm_start(vm, _options = {})
+    group = vm.uid_ems.split('\\').first
```

Comment:
We don't have real objects representing the underlying VM yet in the armrest gem. We're going to refactor https://github.com/ManageIQ/azure-armrest/pull/21 a bit, but for now this will have to do.